packages/twenty-front/src/modules/object-record/object-filter-dropdown/components/ObjectFilterDropdownFilterSelect.tsx

```diff
@@ -53,7 +57,15 @@ export const ObjectFilterDropdownFilterSelect = () => {
   const { getIcon } = useIcons();
 
   const setHotkeyScope = useSetHotkeyScope();
-
+  const {isSelectedItemIdSelector} = useSelectableList(OBJECT_FILTER_DROPDOWN_ID)
+  const isSelectedItemId = (id:string)=> useRecoilValue(isSelectedItemIdSelector(id));
```

Comment:
That's not a common pattern, prefer using a useRecoilCallback and extract it in an external file like useIsListSelectedItem

Reply:
ok got it.